retriever/lib/get_opts.py

```diff
@@ -143,6 +143,7 @@
                                help='write scripts out to a designated directory',
                                nargs='?',
                                const='')
+autocreate_parser.add_argument('-c', help='helps to check for existing data packages', action='store_true')
```

Comment:
Thanks @dassaniansh. The tests still fail and it looks like this line is not well formatted. (Too long)
change from 
`helps to check for existing data packages`
to 
`check if data package already exists` 

Make sure the line length is no more than 90

Reply:
@henrykironde  sir, Is it fine now or I should update anything else